docs/compose/NavBar/README.md

```diff
@@ -140,3 +140,48 @@ Column(modifier.nestedScroll(state)) {
   }
 }
 ```
+
```

Comment:
can you also add the `transparent` screenshot above where the other screenshot tables are?